core/src/epicli/data/common/ansible/playbooks/roles/applications/templates/pgbouncer/configmaps.yml.j2

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: pgbouncer-container-env
+  labels:
+    app: pgbouncer
+  namespace: {{ data.namespace }}
+data:
```

Comment:
I did this that way because i wanted to use more intuitive  and recognizable variables in configuration file. For example i didn't want to use DB_HOST variable like it's used in pgbouncer, because i our case we don't connect to database, we connect to pgpool. So in that case this could be misleading for user. In that case deployment file is more elaborate, but it's easier to read. But i'm open for discussion.